UofT-Frosh-Orientation/orientation-website
# 650 update all graphics branding stuff

## **Contribution:**

- added footer back (not sure why but it was missing from the dev branch)
- updating fonts to Nunito -> branding fonts
- deleted and uploaded new logos (main-logo and favicon)
- changed icon and icon colours for footer to white
- made some small CSS changes to the footer
- updated about us page graphic -> now it's the F!
- changed gradient on home page
- removing some unused class in registration success

### **Issue:**

Closes #650 (still need to optimize image delivery -- taking care of that in another branch) 

### **Accomplishments:**

- updating graphics & some style

### **Visuals:** 

Any related visuals.

## **Details:**

### **Expected Behavior:**

- Everything should compile as regularly, with new font and graphics

### **Testing Steps:**

Steps required to get expected behavior.

- run the front end `yarn start` and navigate through the pages

### **Complications:**

### **Resources:**

Any additional resources used outside this issue.

- [faivcon converter](https://favicon.io/favicon-converter/)
- [converting rgba to hexcode](https://rgbacolorpicker.com/rgba-to-hex)
